docs/source/customize.rst

```diff
@@ -26,6 +26,10 @@ Input Options
   multiple files.
 ``spelling_word_list_filename=['spelling_wordlist.txt','another_list.txt']``
   Same as above, but with several files of correctly spelled words.
+``spelling_ignore_filename=['ignored_file']``
+  List specifying a list of files that should not be checked for spelling.
+  The file extension (e.g. ``.rst``) should not be used when naming a file
+  to ignore.
```

Comment:
What do you think about reusing the code that implements the [exclude_patterns](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-exclude_patterns) option so that the two settings work in the same way?

See the [toctree directive](https://github.com/sphinx-doc/sphinx/blob/3.x/sphinx/directives/other.py#L96) implementation for an example.

If we go this route, the option name should say "patterns" instead of "filenames", so maybe `spelling_exclude_patterns`?

Reply:
I like this method better as it enables excluding entire directories. The PR has been updated to address this.